lib/_stream_writable.js

```diff
@@ -590,6 +591,8 @@ Writable.prototype.end = function(chunk, encoding, cb) {
   // Ignore unnecessary end() calls.
   if (!state.ending)
     endWritable(this, state, cb);
+  else
```

Comment:
I do think this should be 

```suggestion
  else if (typeof cb === 'function')
```

to fix this CI failures?

Reply:
Ah, thanks!